docs/4to5.md

```diff
@@ -0,0 +1,82 @@
+# Jedis 4 to Jedis 5 Breaking Changes
+
+- `StreamConsumersInfo` has been renamed to `StreamConsumerInfo`.
+
+- Return type of `getAgeSeconds()` in `AccessControlLogEntry` has been changed from `String` to `Double`.
+
+- Return type of `bzpopmax(double timeout, byte[]... keys)` has been changed to `List<Object>` (from `List<byte[]>`).
+  - This is a three element list where the last element is a `Double`.
+
+- Return type of `bzpopmin(double timeout, byte[]... keys)` has been changed to `List<Object>` (from `List<byte[]>`).
+  - This is a three element list where the last element is a `Double`.
+
+- Return type of `graphSlowlog(String graphName)` has been changed to `List<List<Object>>` (from `List<List<String>>`).
+
+- The `List<Object> getUnflushedObjectMultiBulkReply()` method has been removed from `Connection`.
+  - `Object getUnflushedObject()` method has been added instead.
+
+- `STREAM_INFO_FULL` in BuilderFactory has been renamed to `STREAM_FULL_INFO`.
+- `STREAM_CONSUMERS_INFO_LIST` in BuilderFactory has been renamed to `STREAM_CONSUMER_INFO_LIST`.
+- `STREAM_AUTO_CLAIM_ID_RESPONSE` in BuilderFactory has been renamed to `STREAM_AUTO_CLAIM_JUSTID_RESPONSE`.
+
+- Following BuilderFactory implementations have been removed:
+  - `BYTE_ARRAY` (use `BINARY`)
+  - `BYTE_ARRAY_LIST` (use `BINARY_LIST`)
+
+<!--- Deprecated in Jedis 4 --->
+
+- `quit()` method has been removed from `ServerCommands` and implementations.
```

Comment:
`quit() method has been removed from `Connection` and ServerCommands implementations`